tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
I am just afraid that these comments won't match the method name any time in the future, and people will get confused by this. Already happened in actual code to me from time to time, and I could imagine that people are spending even less focus on this in the tests.